ecs_windows/cloudwatch.conf

```diff
@@ -0,0 +1,16 @@
+@INCLUDE /ecs_windows/service.conf
+@INCLUDE /ecs_windows/input-forward.conf
+@INCLUDE /ecs_windows/ecs-filters.conf
+
+# In order to use this configuration, following environment variables should be set in container-
+# AWS_REGION -> AWS region to be used for Amazon CloudWatch.
+
+[OUTPUT]
+    Name                cloudwatch_logs
+    Match               out.*
+    region              ${AWS_REGION}
+    log_group_name      fallback-group
+    log_group_template  /aws/ecs/$ecs_cluster.$ecs_task_definition_family
+    log_stream_prefix   task-
+    auto_create_group   On
+    net.dns.resolver    LEGACY
```

Comment:
it might be worth adding a note in the readme about this

Reply:
Added a note about the default CloudWatch configuration.

In the other PR, I am planning to add a note about `net.dns.resolver    LEGACY` in the main README.